templates/_formhelpers.html

```diff
@@ -0,0 +1,14 @@
+{% macro render_field(field) %}
+<div class="formular">
```

Comment:
You should move this `div` outside from the macro and put it around the actual `<form>` tag on the main screen, as described in the first review.